### PR TITLE
Feature 4049 child pare down list view

### PR DESCRIPTION
**Pull request by maccelerated**
In the interest of making Working Waterfronts feature complete, I pared down the ListView. This includes some changes that are not necessary to functionality, but take the code base in a cleaner direction:

- New MapList view and controller replace ListView.
- Marker placement and centering functions added to Map.js view.

A lot of code was cut and pasted, and the way that Google Maps markers are managed has been completely rewritten to remove the use of project-wide app variables.

### CODE REVIEW

File: app/view/Map.js [hunk 1]
```diff
@@ -1,119 +1,138 @@
...
+	addPoint: function (point, id, text, onClick, onClose) {
+		var self	= this;
+		var gMap	= self.map();
+		onClick		= onClick || function () {};
 
-        // this should reset our map center, but it doesn't
-        //  setTimeout(function() {
-        //    WorkingWaterfronts.gMap.panTo(new google.maps.LatLng(44.634115, -124.062796));
-        // }, 100);
+		var html	= '<button onclick="Ext.Viewport.fireEvent(\'mapButton\',\''+id+'\')">' + text + '</button>';
```
Comment (jhcarr) [inline app/view/Map.js]: Please make sure that you comment this line: We are wrestling with scope here - how should we read this line?

File: app/view/Map.js [hunk 2]
```diff
@@ -1,119 +1,138 @@
...
- 
-        var mapPanel = this.down('map');
-        WorkingWaterfronts.gMap = mapPanel.getMap();
+	currentInfoWindow: null,
+	markers: [],
 
-        // console.log('in the initmap function:');
-        // console.log(WorkingWaterfronts);
+	addPoint: function (point, id, text, onClick, onClose) {
+		var self	= this;
```
Comment (jhcarr) [inline app/view/Map.js]: rename to "view" per convention in controller code.

**Comment by jhcarr** [comment]
Just some minor cleanliness tweaks. Otherwise good to go.

Reviewed: 1/12

File: app/view/Map.js (hunk 1 above)
Comment (maccelerated) [inline app/view/Map.js]: Comment added.

File: app/view/Map.js (hunk 2 above)
Comment (maccelerated) [inline app/view/Map.js]: Renamed.

File: app/controller/Home.js [hunk 3]
```diff
@@ -42,7 +36,7 @@ Ext.define('WorkingWaterfronts.controller.Home', {
 		if (toggleValue) {
 			// scope allows callback to use 'this' to get Home controller
 			Ext.device.Geolocation.watchPosition({
-			    frequency	: 3000,
+			    frequency	: 60000,
```
Comment (jhcarr) [inline app/controller/Home.js]: New code review guide says declare units. :D

File: resources/sass/app.scss [hunk 4]
```diff
@@ -22,19 +22,19 @@
 #ext-list-1{
 	height: 50%;
 	top: 50%;
-}	
+}
 #ext-list-2{
 	height: 50%;
 	top: 50%;
-}	
+}
```
Comment (jhcarr) [inline resources/sass/app.scss]: ext-list-1 and ext-list-2 are browser-rendering poorly, check these.

**Pull request merged**